app/models/service_template_ansible_playbook.rb

```diff
@@ -116,4 +106,27 @@ def self.validate_config_info(info)
     info
   end
   private_class_method :validate_config_info
+
+  def job_template(action)
+    resource_actions.find_by!(:action => action.to_s.capitalize).configuration_template
+  end
+
+  def update_catalog_item(options, auth_user = nil)
+    config_info = validate_update_config_info(options)
+    name = options[:name]
+    description = options[:description]
+    [:provision, :retirement, :reconfigure].each do |action|
+      next unless config_info[action]
+      info = config_info[action]
+
+      tower, params = self.class.send(:build_parameter_list, name, description, info)
+
+      params[:manager_ref] = job_template(action).manager_ref
+
+      ManageIQ::Providers::AnsibleTower::AutomationManager::ConfigurationScript.update_in_provider_queue(tower.id, params, auth_user)
```

Comment:
`ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScript`, please update the same for creation.